pkg/network/connection.go

```diff
@@ -239,14 +299,16 @@ func (c *connection) attachEventLoop(lctx context.Context) {
 		},
 
 		onHup: func() bool {
-			log.DefaultLogger.Errorf("[network] [event loop] [onHup] ReadHup error. Connection = %d, Remote Address = %s", c.id, c.RemoteAddr().String())
+			log.DefaultLogger.Errorf("[network] [event loop] [onHup] ReadHup error. Connection = %d, Remote Address = %s", c.id, c.RemoteAddr().String(), c.file.Fd())
```

Comment:
loss format for `c.file.Fd()` .

Reply:
fixed, this onHup func will be removed in future,
currently it's never called